src/components/carousel/index.jsx

```diff
@@ -70,17 +70,20 @@ const Carousel = ({
 	id,
 	label,
 	nextButton,
+	pageCountPhrase,
 	previousButton,
+	showLabel,
 	slidesToShow,
 	fullScreenShowButton,
 	fullScreenMinimizeButton,
 	enableFullScreen,
 	...rest
 }) => {
+	// slidesToShow is a number of slides to show at once
 	const [slide, setSlide] = useState(slidesToShow);
 	const [position, setPosition] = useState(0);
 	const [isFullScreen, setIsFullScreen] = useState(false);
```

Comment:
I am curious....I tried this out on storybook and it's complete computer fullscreen vs just a "fullscreen" browser window/modal.  Are we sure we want a true full screen presentation?  For example, USNews has a "Full screen" for its gallery, but it's a full browser width and height modal that doesn't take over the user's screen:

![image](https://user-images.githubusercontent.com/833461/163845272-3ea5310a-8c85-42c2-9e4b-eb9d5dbc8d17.png)

Fullscreen computer display seems like it should be reserved for video or games you know, maybe not so much for a gallery.

Reply:
yep, this is verified from product and Jenae as the desired behavior with full screen api. I put this is in the notes as well as the ticket in a previous pr #64 . @visionbegin

Reply:
We'll probably need a "lightbox mode" or something for that sort of expansion.

Reply:
Yeah. I appreciate the question and research from a product side @visionbegin . In my opinion, I prefer using the actual full screen of the window for appreciating the pictures (like videos) as big as possible when I do "expand" a gallery. And Jenae agreed from a product-perspective.

From a technical perspective, I like this because it's using built-in browser technologies. As a result, we save bundle size on package installation (giving us more room for motion libs @visionbegin ). Also, we provide a native experience to users on whatever operating system and browser they are on. 

Also, I wasn't sure where to begin to refactor the existing engine-theme-sdk implementation that has been ported from another library to typescript. In this way, we are starting fresh with our browser support.

Reply:
Cool, thanks for the explanation....The argument about saving bundle size make a ton of sense!